From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: MichaelSimons <msimons@microsoft.com>
Date: Wed, 20 Jul 2022 23:49:52 +0000
Subject: [PATCH] Disable prebuilt baseline

This patch is needed until the previous source-build artifacts are updated.
The current source-build artifacts have some prebuilt references which cause issues.
---
 eng/SourceBuildPrebuiltBaseline.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/eng/SourceBuildPrebuiltBaseline.xml b/eng/SourceBuildPrebuiltBaseline.xml
index c9c5565..6fb38a5 100644
--- a/eng/SourceBuildPrebuiltBaseline.xml
+++ b/eng/SourceBuildPrebuiltBaseline.xml
@@ -1,6 +1,7 @@
 <UsageData>
   <IgnorePatterns>
     <UsagePattern IdentityGlob="Microsoft.SourceBuild.Intermediate.*/*" />
+    <UsagePattern IdentityGlob="*/*" />
   </IgnorePatterns>
   <Usages>
     <Usage Id="System.Text.Encoding.CodePages" Version="6.0.0" />
